### ISSUE TITLE
Broker fields too short

### ISSUE THREAD

**Issue by pjz** [blinktrade/bitex#23]
Currently Broker has:
```                                                             
phone_number_1        = Column(String(15))
phone_number_2        = Column(String(15))
skype                 = Column(String(30))
email                 = Column(String(15))
```
which strike me as a bit... short.  Your demo data (in `bootstrap/demo.ini`) thinks so too, since it overflows the email field (`len("user@blinktrade.com")` is > 15).  

My suggested fix is to make phone numbers 30 characters and skype and email 254 characters. I could  submit a PR, but I don't know how you guys are doing schema change management so didn't want to trample on it and risk messing with someone's live database.

**Comment by pinhopro** [comment]
Good point.  Since I am using sqlite3, I didn't catch those. I will be replacing those. 

Thanks
